site/docs/v1/tech/core-concepts/authentication-authorization.adoc

```diff
@@ -45,19 +45,21 @@ include::docs/src/json/login/jwt.json[]
 
 === Authorization and Securing Your Application
 
-These concepts are critical to application security. 
+These concepts are critical to application security.
 
 If you are utilizing the JWT to authorize a user to your application, you must do more than just ensure the JWT has a valid signature and is not expired. You must also ensure the JWT has provided adequate claims to the user's authorization.
 
+If you enable "Require Registration" on an application, the JWT won't be provided until the user is registered for this application.
+
 *Checking the JWT signature and expiration are only a part of the story.*
 
 The `aud` claim identifies the context of the request, in other words who is this JWT for: a Payroll application, a mobile application, etc. The presence of the `applicationId` and `roles` claims identifies the User's registration (authorization) and access (roles) to the requested resource identified by the `aud` claim.
 
 === An Example
 
-Say Bob has access to applications A, B and C configured in FusionAuth, and Lisa has access to applications C, D and E. Assume there's also an application F to which neither Bob nor Lisa is registered. 
+Say Bob has access to applications A, B and C configured in FusionAuth, and Lisa has access to applications C, D and E. Assume there's also an application F to which neither Bob nor Lisa is registered.  Note that for this example to work all the applications A thru F have "Require Registration" disabled (set to `false`).
```

Comment:
@mooreds I added your suggestion above L52 and reiterated the idea here L60 to help enforce the idea of how "Require Registration" works.  Let me know what you think.

Reply:
> Require registration

(lc registration)

Reply:
Fixed